webpack.config.common.js

```diff
@@ -26,38 +26,39 @@ module.exports = {
     path: path.join(__dirname, 'build'),
     publicPath: '/',
     filename: 'client.js',
-    chunkFilename: '[name].[chunkhash].js',
+    // chunkFilename: '[name].[chunkhash].js',
   },
   optimization: {
-    chunkIds: 'named',
```

Comment:
We should move it into 'webpack.config.prod.js'.
WDYT?

Reply:
Just didn't revert temporary changes.